app/app.go

```diff
@@ -346,20 +328,20 @@ func NewAxelarApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 	app.stakingKeeper = stakingK
 
 	// add capability keeper and ScopeToModule for ibc module
-	app.capabilityKeeper = capabilitykeeper.NewKeeper(appCodec, keys[capabilitytypes.StoreKey], memKeys[capabilitytypes.MemStoreKey])
+	capabilityKeeper := capabilitykeeper.NewKeeper(appCodec, keys[capabilitytypes.StoreKey], memKeys[capabilitytypes.MemStoreKey])
 
 	// grant capabilities for the ibc and ibc-transfer modules
-	scopedIBCK := app.capabilityKeeper.ScopeToModule(ibchost.ModuleName)
-	scopedTransferK := app.capabilityKeeper.ScopeToModule(ibctransfertypes.ModuleName)
+	scopedIBCK := capabilityKeeper.ScopeToModule(ibchost.ModuleName)
+	scopedTransferK := capabilityKeeper.ScopeToModule(ibctransfertypes.ModuleName)
 
 	// Create IBC Keeper
-	app.ibcKeeper = ibckeeper.NewKeeper(
+	ibcKeeper := ibckeeper.NewKeeper(
```

Comment:
```suggestion
	ibcK := ibckeeper.NewKeeper(
```

Reply:
the fully initialized ibcK is already initialized on line 409. I'll keep this in mind and this assignment should likely get eliminated in subsequent refactorings, so I'm not going to change it just yet